web/src/sdk/links/retryLink.js

```diff
@@ -1,20 +1,22 @@
 import {RetryLink} from "apollo-link-retry";
 import { getOperationDefinition } from "apollo-utilities";
 
-export const retryOnErrorLink = (numOfAttempts, firstAttempAfter) => (
+export const retryOnErrorLink = (numOfAttempts = 5, firstAttempAfter = 2000) => (
   new RetryLink({
-    delay: {initial: firstAttempAfter || 1000},
+    delay: {initial: firstAttempAfter, jitter: false},
     attempts: (count, operation, error) => {
       const { operation: operationType } = getOperationDefinition(operation.query);
       const isMutation = operationType === 'mutation';
-      console.info(`retryOnErrorLink: {      
-        numOfAttempts: ${count}, 
+      if(!isMutation) { return }
```

Comment:
yes that solves the problem but why not
```suggestion
      if(isMutation) {
```

Reply:
I think at the end of the day it will be the same because of the other return we need.